RELEASE.md

```diff
@@ -1,3 +1,75 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Cloud TPU:
+  * Experimental tf.data integration for Keras on Cloud TPUs.
+* Add C functions for querying kernels
+* Add C++ functions for querying kernels
+* Add Python functions for querying kernels
+
+## Breaking Changes
```

Comment:
I think we'll need to add cudnn 7.2 changes, maybe something like:
```
Prebuilt binaries are now (as of TensorFlow 1.11) built against cuDNN 7.2. See updated install guides: [Installing TensorFlow on Ubuntu](https://www.tensorflow.org/install/install_linux#tensorflow_gpu_support)
```
Also @tfboyd .

Reply:
Added in the upcoming push, thanks!